src/platform/utilities/toggles/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import environments from 'platform/utilities/environment';
+
+function makeEnvironmentToggleValues(env = environments) {
+  return {
+    develop: env.isDev(),
+    localhost: env.isLocalhost(),
+    notProduction: !env.isProduction(),
```

Comment:
decided to throw the environment toggles on here as a test. Also `notProduction` is so commonly used, I gave it an explicit key. I think as a guideline our toggles should avoid negatives like this.